src/angular-multi-select-utils.js

```diff
@@ -74,7 +76,7 @@ angular_multi_select_utils.factory('angularMultiSelectUtils', [
 			for (var i = 0; i < arr.length; i+= 2) {
 				var value = arr[i + 1];
 
-				if (value.match(/^([0-9\.]*)$/gi) !== null) {
+				if (value && value.match(/^([0-9\.]*)$/gi) !== null) {
```

Comment:
This shouldn't be required after you fix the other function (check my previous comment).